packages/earn-controller/src/EarnController.ts

```diff
@@ -219,25 +217,12 @@ export function getDefaultEarnControllerState(): EarnControllerState {
 
 // === MESSENGER ===
 
-/**
- * The action which can be used to retrieve the state of the EarnController.
- */
-export type EarnControllerGetStateAction = ControllerGetStateAction<
-  typeof controllerName,
-  EarnControllerState
->;
-
-/**
- * All actions that EarnController registers, to be called externally.
- */
-export type EarnControllerActions = EarnControllerGetStateAction;
```

Comment:
It's standard for all controllers to export an `*Actions` type, even if nothing is there. What are your thoughts on keeping this but setting the right hand side to `never`?

Reply:
Got it, I think I can just completely revert this change then rather than setting it to `never`?